server.go

```diff
@@ -19,6 +20,32 @@ func fallback(w http.ResponseWriter, r *http.Request, reason string) {
 	http.Redirect(w, r, location, 302)
 }
 
+func getRedirect(txt []string, url string) (*Redirect, error) {
+	catch_alls := make(map[string]*Config)
```

Comment:
Yep, I understand that we're not really gaining much since there's no guaranteed order returned by the DNS lookup. However, I do want to preserve the order that is returned to make debugging a bit simpler.